src/main/java/net/mcreator/blockly/java/blocks/GetVariableBlock.java

```diff
@@ -61,25 +62,27 @@ public GetVariableBlock() {
 				if (scope.equals("global") && !master.getWorkspace().getVariableElements().stream()
 						.map(VariableElement::getName).collect(Collectors.toList()).contains(name)) {
 					master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-							"Variable get block is bound to a variable that does not exist. Remove this block!"));
+							L10N.t("blockly.errors.variables.invalid_var", L10N.t("blockly.block.get_var"),
+									L10N.t("blockly.errors.remove_block"))));
 					return;
 				} else if (master instanceof BlocklyToProcedure && scope.equals("local")
 						&& !((BlocklyToProcedure) master).getLocalVariables().stream().map(VariableElement::toString)
 						.collect(Collectors.toList()).contains(name)) { // check if local variable exists
 					master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-							"Variable get block is bound to a local variable that does not exist. Remove this block!"));
+							L10N.t("blockly.errors.variables.invalid_local_var", L10N.t("blockly.block.get_var"),
+									L10N.t("blockly.errors.remove_block"))));
 					return;
 				} else if (scope.equals("local") && !(master instanceof BlocklyToProcedure)) {
 					master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-							"This editor does not support local variables!"));
+							L10N.t("blockly.warnings.variables.local_scope_unsupported")));
 					return;
 				} else if (scope.equalsIgnoreCase("local")) {
 					List<StatementInput> statementInputList = master.getStatementInputsMatching(
 							statementInput -> statementInput.disable_local_variables);
 					if (!statementInputList.isEmpty()) {
 						for (StatementInput statementInput : statementInputList) {
 							master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-									"Statement " + statementInput.name + " does not support local variables."));
+									L10N.t("blockly.errors.variables.no_local_scope.statement", statementInput.name)));

```

Comment:
Where is translation for this key?

Reply:
It ia the line 2774, but I accidently replaced the last . by a _.